services/user.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
According to the checklist, you should use `get_user_model()` instead of directly importing `User` from `db.models`. This ensures compatibility with custom user models.